File: 27-RemoveElement/27-RemoveElement.java
// Last updated: 28/08/2025, 15:45:38
class Solution {
    public int removeElement(int[] nums, int val) {
        if(nums.length == 0) return 0;
        int size = 0; 
        for(int i = 0; i < nums.length; i++) {
            if(nums[i] != val) {
                nums[size++] = nums[i];
            }
        }
        return size;
    }
}

File: 27-RemoveElement/27-RemoveElement.py
# Last updated: 20/11/2025, 21:23:27
class Solution:
    def removeElement(self, nums: List[int], val: int) -> int:
        k = 0

        for i in range(len(nums)):
            if nums[i] != val:
                nums[k] = nums[i]
                k += 1

        return k